### PR TITLE
Fix accessibility issues

### LINKED ISSUE #194: Lighthouse Accessibility regressions on /parts
Running lighthouse on https://react-commerce-ifixit.vercel.app/store/parts and https://me.cominor.com/Store/Parts shows there has been some regression on accessibility.

This accessibility issue  does not exist  on the current page: 
image

Note: other pages have higher accessibility scores than their current counterparts.

### PR DESCRIPTION
**Pull request by dhmacs**
fixes #194 

## QA

1. Visit the Vercel PR preview (replace `vercel.app` with `govinor.com`)
2. Visit parts product list page
3. Use lighthouse to verify that accessibility is > 90

### CODE REVIEW

File: frontend/components/common/Layout/User.tsx [hunk 1]
```diff
@@ -113,34 +101,69 @@ export function UserMenu() {
...
+         src={user.thumbnail || undefined}
+         icon={
+            <NoUserIcon
+               transition={`color ${DEFAULT_ANIMATION_DURATION_MS}`}
+               _hover={{
+                  color: 'brand.300',
+               }}
+            />
+         }
+         bg={user.thumbnail == null ? undefined : 'transparent'}
```
[the remark this answers is not in the document]
Comment (danielbeardsley) [reply frontend/components/common/Layout/User.tsx:131]: Oh... just read the docs, `icon` will be used if `src` isn't provided.

**Comment by danielbeardsley** [comment]
CR :+1:

File: frontend/components/common/Layout/User.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (dhmacs) [reply frontend/components/common/Layout/User.tsx:131]: That code is like this to fix this issue 👉 https://github.com/iFixit/react-commerce/issues/202

**Review by ianrohde — APPROVED** [review]
CR ⚪

**Pull request merged**